### ISSUE TITLE
Possible to add domain end label?

### ISSUE THREAD

**Issue by ministe2003** [pandameister/chartjs-chart-radial-gauge#7]
New to Chart.js and to this plugin, its also the first time I've had to work with a canvas, so its all new to me!

I'm liking this gauge plugin, but wondered if its possible to display the domain end value?  I'm using this gauge (like most will be, I expect) to show progress towards a target, so whilst I like being able to show the actual value in the centre, I'd like to show the target too so everyone knows where they're working towards.  Something like this:

<img width="226" alt="Screenshot 2020-02-18 at 16 40 30" src="https://user-images.githubusercontent.com/19466487/74757421-e11e6580-526d-11ea-8fd9-141e75159bca.png">

Doesn't look like this is available right now, is it something that could be added as an option?

Thanks!

**Comment by pandameister** [comment]
@ministe2003 yup, it's not available right now. I'm bit tied up with my day job right now, but I'll try to get to it soon, or you could submit a PR for review, thanks
